### ISSUE TITLE
[2.0.x] Model::find(array("id_column=0")) causes PDO exception

### ISSUE THREAD

**Issue by jimjim2a** [phalcon/cphalcon#10553]
Hello,

I'm trying to migrate from Phalcon 1.3.4 to Phalcon 2.0.4, and I'm facing this ORM issue : 

```<?php
$di = new \Phalcon\DI\FactoryDefault();

class X extends \Phalcon\DI\Injectable
{
    public function x()
    {
        $users = User::find(array("id_parent=0"));
    }
}

$x = new X;
$x->setDI($di);
$x->x();
```

This produces this PDO query :
```
SELECT "user"."id, "user"."id_parent", "user"."name" FROM "public"."user" WHERE "user"."id_parent" = 
```
It seems that it is assigning 0 as value that is causing the problem.

If I replace 0 by '0', the issue does not occur anymore.

Thanks for your help.

**Comment by andresgutierrez** [comment]
Fixed in the 2.0.x branch

**Comment by jimjim2a** [comment]
Fix confirmed !

Thx !
